### ISSUE TITLE
Review and Test all visible focus related tickets potentially fixed with GH-6052

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#6333]
We have over 20 tickets related to visible focus issues that might have been fixed when merging GH-6052.

- Add specific browser screenshot to ticket.
- Add theme-calydon label to all tickets needing calydon to be active.
- Add fixed in GH-6052 reference
- Add link to testing instructions if it's a Safari bug

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 1 21 37 PM](https://user-images.githubusercontent.com/15113219/73564003-af636d00-442c-11ea-9af3-9f6d69e02148.png)
![Screen Shot 2020-01-31 at 1 21 47 PM](https://user-images.githubusercontent.com/15113219/73564005-af636d00-442c-11ea-9369-c7bb5bbd5fc5.png)

**Comment by developpeurweb** [comment]
33 tickets reviewed and tested.
This was a task control ticket, **nothing to QA** for this one specifically.
